errors/errors.go

```diff
@@ -0,0 +1,128 @@
+package errors
+
+import "errors"
+
+type errorType int
+
+const (
+	_ErrNotFound errorType = iota + 1
+	_ErrConflict
+	_ErrConnection
+	_ErrAlreadyExists
+	_ErrGeneric
+	_ErrWatch
+	_ErrUnauthorized
+	_ErrForbidden
```

Comment:
If this follows the Uber-Style-Guide, it is stated that unexported errors should start with small `err` instead of `_` :

https://github.com/uber-go/guide/blob/master/style.md#error-naming